.travis.yml

```diff
@@ -5,6 +5,8 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
+  - 2.7
+  - 3.0
```

Comment:
[note] Should we update also `required_ruby_version` because the CI tests `>= 2.4`?

```ruby
gem.required_ruby_version = '>= 2.4.0' 
```

https://github.com/sourcelevel/faraday-http-cache/blob/5c055e9129de770ad5d964d86ecb3e2ef447678d/faraday-http-cache.gemspec#L17

Reply:
you probably want to add an exclude below so that tests don't run with faraday 0.x and ruby 3

also, I'm surprised travis still works on opensource project... probably time to swich to github actions